torch/csrc/jit/passes/xnnpack_rewrite.cpp

```diff
@@ -246,6 +270,22 @@ void runCanonicalOptimizations(script::Module& module) {
 
 } // namespace
 
+void transformConv1dToConv2d(std::shared_ptr<Graph>& graph) {
+  // Replace _convolution with conv1d and conv2d
+  graph_rewrite_helper::replaceConvolutionWithAtenConv(graph);
+  replaceConv1dWithConv2d(graph);
+}
+
+void transformConv1dToConv2d(script::Module& module) {
+  for (auto& method : module.get_methods()) {
+    auto graph = method.graph();
```

Comment:
`auto&` to avoid refcount bump.

Reply:
I did try it, but it raised an error, so I just went for `auto`.